### PR TITLE
Added more css and html

### PR DESCRIPTION
**Pull request by askiefer**
This PR redoes a bit of the html for the latest mockup and adds html to the side panel.

I'll add a function that aligns the body "info" panel with the bottom of the side panel once the other graphs are in (which will push the bottom panel down some).

screen shot 2017-03-21 at 9 28 33 am

### CODE REVIEW

**Comment by askiefer** [comment]
@thfield @eayoungs Here are the ids of the dynamic variables in the app. Most of them are pretty self-explanatory, and in general I went for clarity over brevity, though happy to change any we'd like:

- .building-name
- .building-type-sq-ft (the square footage of all the building types in the category, i.e. 150,000-200,000 sq ft)
- .building-type
- #building-street-address
- #building-city-address
- #building-ranking (an individual building's ranking based on our calculation of the z-score)
- #total-building-type (the total number of buildings of a particular type, i.e. 250 office)
- #energy-star-score-radial 
- #building-energy-star-score (the E star score of the individual building)
- #median-energy-star-score (the median E star score of all of the buildings of the particular type)
- #energy-star-score-histogram (``chart 1``)
- #building-eui (EUI of the specific building)
- #eui-relative-ranking (EUI ranking of the specific building relative to other buildings in the category, i.e. lower, higher, or similar to, ``Hotel Zetta has an EUI lower than the national median for hotels.``)
- #eui-stacked-bar (``chart 2``)
- #building-ghg-emissions (the specific building's GHG emissions)
- #eui-histogram (``chart 3``)

File: dashboard/dashboard.html [hunk 1]
```diff
@@ -113,60 +113,72 @@
...
+		          		<div class="col-xs-3">
+		          			<div class="row center">
+		          				<span class="circle-chart" id="energy-star-score-radial"></span>
+		          				<span class="ranking-num" id="building-energy-star-score">43</span><br><span class="ranking-units">out of 100</span>
+		          			</div>
+		          		</div>
+			          	<div class="col-xs-7">
 			          		<div class="ranking-text">
-			          		In 2015, Hotel Zetta earned an ENERGY STAR score of 43. The hotel median score is 50.
+			          		In 2015, <span id="building-name">Hotel Zetta earned an ENERGY STAR score of <span id="building-energy-star-score">43</span>. The <span id="building-type">hotel</span> median score is <span id="median-energy-star-score">50</span>.
```
Comment (thfield) [inline dashboard/dashboard.html]: some of these get repeated on the page, like `building-name` and `building-type`, so should be `class="building-name"` instead

File: dashboard/dashboard.html [hunk 2]
```diff
@@ -113,60 +113,72 @@
...
+		          		<div class="col-xs-3">
+		          			<div class="row center">
+			          			<span class="speedometer-chart" id="speedometer-chart"></span>
+			          			<span id="building-eui" class="ranking-num">59.9</span><span class="ranking-units">kbtu/ft<sup>2</sup></span>
+			          		</div>
+		          		</div>
+			          	<div class="col-xs-7">
 			          		<div class="ranking-text">
-			          		Hotel Zetta has an EUI lower than the national hotel median for hotels.
+			          		<span id="building-name">Hotel Zetta</span> has an EUI <span id="eui-relative-ranking">lower</span> than the national <span id="building-type">hotel</span> median for hotels.
```
Comment (thfield) [inline dashboard/dashboard.html]: `...than the national median for <span id="building-type">hotel</span>s.`

**Review by thfield — COMMENTED** [review]
just looked through code, didn't pull and run locally yet

File: dashboard/dashboard.html (hunk 1 above)
Comment (thfield) [reply dashboard/dashboard.html]: also, `In 2015, <span id="building-name">Hotel Zetta</span>`

Comment (askiefer) [reply dashboard/dashboard.html]: Totally @thfield, I changed them and lumped all of the classes together in the list above. I'll recheck this evening to make sure I included all of the repeats.

**Comment by askiefer** [comment]
@eayoungs @thfield just FYI I just pushed the latest html/css changes, also includes the icons / svgs Juliana created

**Pull request merged**